docs/uvcal_tutorial.rst

```diff
@@ -475,7 +478,14 @@ each file will be read in succession and added to the previous.
   >>> settings_testfiles = [os.path.join(DATA_PATH, f) for f
   ...                       in ['fhd_cal_data/1061316296_settings.txt',
   ...                           'fhd_cal_data/set2/1061316296_settings.txt']]
-  >>> fhd_cal.read_fhd_cal(cal_testfiles, obs_testfiles, settings_file=settings_testfiles)
+  >>> layout_testfiles = [
+  ...   os.path.join(DATA_PATH, f) for f in [
+  ...     'fhd_cal_data/1061316296_layout.sav', 'fhd_cal_data/1061316296_layout.sav'
+  ...   ]
+  ... ]
```

Comment:
I feel like this would be much simpler as an explicit list:

```python
layout_testfiles = [
    os.path.join(DATA_PATH, 'fhd_cal_data/1061316296_layout.sav'), 
    os.path.join(DATA_PATH, 'fhd_cal_data/1061316296_layout.sav'),
]
```